testdata/testing.txt

```diff
@@ -1,10 +1,10 @@
---- FAIL: TestBar
+--- FAIL: TestBar (0.00s)
```

Comment:
This seems a bit concerning here. Do we somehow disable the timer, or does this rely on the tests running quickly?

Reply:
Good catch. 

Fixed.  main_test.go now rewrites the received bytes.